app/scripts/TrackRenderer.jsx

```diff
@@ -949,21 +1282,23 @@ class TrackRenderer extends React.Component {
     this.applyZoomTransform(false);
   }
 
-  updateMetaTracks() {
+  /** @param {unknown} _unused */
+  updateMetaTracks(_unused) {
     // Nothing
   }
 
+  /** @param {Array<TrackDefinition>} newTrackDefs */
   updateExistingTrackDefs(newTrackDefs) {
-    for (let i = 0; i < newTrackDefs.length; i++) {
```

Comment:
Changes not totally necessary here, but keeping track of `i` in the for loop is a bit mindbending.